### DISCUSSION TITLE
Teacher Benefits Denied

### DISCUSSION THREAD

**Discussion by mlembo34** [community/community#54034] [GitHub Education]
### Hi everyone,

I applied for the teacher benefits several times and have been denied. Each time is states that I need to fill out my user bio. I have done that, and it still denies me. Can anyone help with this?

![GitHub](https://user-images.githubusercontent.com/132035496/235305741-b41a46a3-9138-4f31-82da-bd67cfd4b01c.jpg)

**Comment by samus-aran** [comment]
Hi @mlembo34 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈
